src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.service.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private final UserDao userDao;
+
+    @Autowired
```

Comment:
i think there is enough to use field injection (remove constructor)

Reply:
fixed